tests/docarray/test_save_load.py

```diff
@@ -14,6 +16,7 @@ def test_save_load(tmpfile):
     # need release the resource
     del save_da
 
+    os.system('rm {}'.format(os.path.join(tmpfile, 'metas.db')))
```

Comment:
@jemmyshin why do we need to remove `meta.db` manually? Shouldn't the deleting be performed in `del save_da`? And please always use Python API, rather than os subprocess. It is not pythonic and professional.